tests/sentry/api/endpoints/test_organization_activity.py

```diff
@@ -23,3 +24,44 @@ def test_simple(self):
 
         response = self.get_success_response(org.slug)
         assert [r["id"] for r in response.data] == [str(activity.id)]
+
+    def test_paginate(self):
+        group = self.group
+        org = group.organization
+        project_2 = self.create_project()
+        group_2 = self.store_event(
+            data={
+                "timestamp": iso_format(before_now(minutes=1)),
+                "tags": {"group_id": "group-2"},
+            },
+            project_id=project_2.id,
+        ).group
+
+        activity = Activity.objects.create(
+            group=group,
+            project=group.project,
+            type=Activity.NOTE,
+            user=self.user,
+            data={"text": "hello world"},
+        )
+        activity_2 = Activity.objects.create(
+            group=group_2,
+            project=group_2.project,
+            type=Activity.NOTE,
+            user=self.user,
+            data={"text": "hello world 2"},
+        )
+        activity_3 = Activity.objects.create(
+            group=group,
+            project=group.project,
+            type=Activity.NOTE,
+            user=self.user,
+            data={"text": "hello world 3"},
```

Comment:
Should you be more explicit about the datetimes on these records so that the order can't wobble?

Reply:
I've relied on creation order of models in a few places and haven't seen the tests go unstable, so I'm ok with leaving it for the moment.